### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by AndriiBardin**

### CODE REVIEW

**Review by yurwing — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1 +1,30 @@
-# hibernateHW
+## Cinema Ticket Service
+
+This project was developed according to Solid principles using N-layer architecture with authorization and authentication included.
+The following layers were created **DB layer,** **DAO layer,** **Service layer,** **Controllers layer** and **View layer**
```
Comment (KseniiaMakarova) [inline README.md]: where is the view layer in your project?

File: README.md [hunk 2]
```diff
@@ -1 +1,30 @@
...
+This project was developed according to Solid principles using N-layer architecture with authorization and authentication included.
+The following layers were created **DB layer,** **DAO layer,** **Service layer,** **Controllers layer** and **View layer** 
+
+On the following diagram the relations between the different enteties is represented.
+
+![img](https://user-images.githubusercontent.com/74858422/109378729-8d614600-78dd-11eb-9b1f-9528a95e77b6.png)
+
+Brief description of the app functions and roles of users:
+
+User allowed :
```
Comment (KseniiaMakarova) [inline README.md]:
```suggestion
User is allowed to:
```

File: README.md [hunk 3]
```diff
@@ -1 +1,30 @@
...
+
+To run the application:
+
+- Install JDK (https://www.oracle.com/java/technologies/javase-downloads.html)
+- Download and install servlet container (Tomcat)
+- Download and install Database (MySQL) and set up the connection to DB in db.properties file
+- There are 2 user roles "USER" and "ADMIN". To log in as admin use "admin" as login "admin" as password
+- To test the application suggest using Postman or other API-testing applications
+
+### written by Andrii Bardin
```
Comment (KseniiaMakarova) [inline README.md:40]: what technologies did you use?

File: README.md [hunk 4]
```diff
@@ -1 +1,30 @@
...
+- View the orders history
+
+Admin has access to POST, GET, PUT and DELETE methods for almost all endpoints, except user-exclusive endpoints (/shopping_cart and /orders)
+
+To run the application:
+
+- Install JDK (https://www.oracle.com/java/technologies/javase-downloads.html)
+- Download and install servlet container (Tomcat)
+- Download and install Database (MySQL) and set up the connection to DB in db.properties file
+- There are 2 user roles "USER" and "ADMIN". To log in as admin use "admin" as login "admin" as password
```
Comment (KseniiaMakarova) [inline README.md]:
```suggestion
- There are 2 roles - "USER" and "ADMIN". To log in as admin use "admin" as login and "admin" as password
```

**Review by KseniiaMakarova — CHANGES_REQUESTED** [review]
let's merge PRs that need to be merged, and delete all redundant branches from GitHub

File: README.md [hunk 5]
```diff
@@ -1 +1,40 @@
...
+- MySQL
+- Jackson
+- Tomcat
+
+**To run the application:**
+
+- Install JDK (https://www.oracle.com/java/technologies/javase-downloads.html)
+- Download and install servlet container (Tomcat)
+- Download and install Database (MySQL) and set up the connection to DB in db.properties file
+- There are 2 user roles - "USER" and "ADMIN". To log in as admin use "admin" as and login "admin" as password
```
Comment (KseniiaMakarova) [inline README.md:37]:
```suggestion
- There are 2 user roles - "USER" and "ADMIN". To log in as admin use "admin" as login and "admin" as password
```

**Review by KseniiaMakarova — APPROVED** [review]

**Pull request merged**